src/libs/uploads.ts

```diff
@@ -105,12 +162,17 @@ export const useUploader = () => {
     if (!signal.aborted) {
       dispatch({ action: 'finish' })
     }
-  }, [signal])
+
+    // useCancelable will call abort when unmounted. After files are uploaded,
+    // we no longer care about that method of cancellation. Catch here to
+    // avoid an unhandled promise rejection.
+    uploadCancelled.catch(() => {})
```

Comment:
It seems the app was handling this unhandled Promise rejection somewhere, but it was causing unit tests to fail (and the test runner to crash).